### PR TITLE
quickfixed PHP-Warning Message for inlineLink

### PR DESCRIPTION
**Pull request by wakka90**
return NULL-value for invalid inputs

for more information:
#5

### CODE REVIEW

File: vendor/erusev/parsedown-extra/ParsedownExtra.php [hunk 1]
```diff
@@ -302,6 +302,10 @@ protected function inlineFootnoteMarker($Excerpt)
     protected function inlineLink($Excerpt)
     {
         $Link = parent::inlineLink($Excerpt);
+        
+        if ( is_null($Link) || is_null($Link['extent'])) {
+            return null;
+        }
```
Comment (Chaosmeister) [inline vendor/erusev/parsedown-extra/ParsedownExtra.php:308]: I think we probably should just update ParsedownExtra at this point.

Comment (creecros) [reply vendor/erusev/parsedown-extra/ParsedownExtra.php:308]: ya, I've had to do it before, it works.

Comment (creecros) [reply vendor/erusev/parsedown-extra/ParsedownExtra.php:308]: or you mean PR in that repo....if it's still active, would definitely make it easier.

Comment (creecros) [reply vendor/erusev/parsedown-extra/ParsedownExtra.php:308]: last merge I see was Oct 2021. might be wishful thinking

Comment (Chaosmeister) [reply vendor/erusev/parsedown-extra/ParsedownExtra.php:308]: looks like this issue was already fixed in there though.

Comment (creecros) [reply vendor/erusev/parsedown-extra/ParsedownExtra.php:308]: then we can run composer update. might solve #3 as well.

Comment (creecros) [reply vendor/erusev/parsedown-extra/ParsedownExtra.php:308]: totally forgot to do this

**Pull request closed without merging**